Refresh.GameServer/Types/Levels/GameMinimalLevelResponse.cs

```diff
@@ -82,6 +86,7 @@ public class GameMinimalLevelResponse : IDataConvertableFrom<GameMinimalLevelRes
             MinPlayers = level.MinPlayers,
             TotalPlayCount = level.TotalPlayCount,
             UniquePlayCount = level.UniquePlayCount,
+            AverageRating = 3, //TODO
```

Comment:
0 is a better default imo, I think it shows that there are no ratings to the user in a simpler way
```suggestion
            AverageRating = 0, //TODO
```